qtawesome/icon_browser.py

```diff
@@ -92,9 +92,17 @@ def __init__(self):
         self._lineEdit.setFocus()
 
         geo = self.geometry()
-        desktop = QtWidgets.QApplication.desktop()
-        screen = desktop.screenNumber(desktop.cursor().pos())
-        centerPoint = desktop.screenGeometry(screen).center()
+
+        # QApplication.desktop() has been removed in Qt 6.
+        # Instead, QGuiApplication.screenAt is supported in Qt 5.10 or later.
+        try:
+            desktop = QtWidgets.QApplication.desktop()
+            screen = desktop.screenNumber(desktop.cursor().pos())
+            centerPoint = desktop.screenGeometry(screen).center()
+        except:
```

Comment:
```suggestion
        except AttributeError:
```
Since the exception thrown by calling `QtWidgets.QApplication.desktop()` should be an `AttributeError`